options.js

```diff
@@ -88,88 +94,123 @@ function updateShortcutInputText(inputId, keyCode) {
   document.getElementById(inputId).keyCode = keyCode;
 }
 
+function updateCustomShortcutInputText(inputItem, keyCode) {
+  inputItem.value = keyCodeAliases[keyCode] || String.fromCharCode(keyCode);
+  inputItem.keyCode = keyCode;
+}
+
+function add_shortcut() {
+  var html = '<select class="customDo">' +
```

Comment:
Let's use template literals here, it'll make it much easier to maintain: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals